docs/repository_rules.md

```diff
@@ -7,62 +7,26 @@ Public API for repository rules
 ## rb_bundle
 
 <pre>
-rb_bundle(<a href="#rb_bundle-name">name</a>, <a href="#rb_bundle-env">env</a>, <a href="#rb_bundle-gemfile">gemfile</a>, <a href="#rb_bundle-repo_mapping">repo_mapping</a>, <a href="#rb_bundle-srcs">srcs</a>)
+rb_bundle(<a href="#rb_bundle-toolchain">toolchain</a>, <a href="#rb_bundle-kwargs">kwargs</a>)
 </pre>
 
 
-Installs Bundler dependencies and registers an external repository
```

Comment:
Shall we move this documentation for the rule to the macro?

Reply:
My solution for this, at least with current stardoc, is to document both. Described at https://github.com/bazelbuild/stardoc/issues/98#issuecomment-1699137341